docker/README.md

```diff
@@ -0,0 +1,23 @@
+## Running the web tool with maurolepore/r2diirunner-with-packages
+
+[Video](https://asciinema.org/a/364647)
+
+* Ensure your working directory is PACTA_analysis
+
+* Update local repos to the latest commit on the main branch:
+
+```bash
+./bin/update-repos
+```
+
+* Run the web tool in a single step
+
+```bash
+./bin/run-web-tool
+```
+
+* Or only start the docker container (e.g. to debug)
+
+```bash
+./bin/start-container
```

Comment:
likewise, is this an existing file?

Reply:
See https://github.com/2DegreesInvesting/PACTA_analysis/blob/docker/docker/2diirunner-with-packages/README.md

Reply:
`./bin/start-container` is not mentioned in the README file you point to

in any case, I see that file exists in master now, so it must have been added at some point before I had last updated